.circleci/config.yml

```diff
@@ -1,54 +1,56 @@
 version: 2
-# Required secret environment variables
-# AWS_ACCESS_KEY_ID
-# AWS_SECRET_ACCESS_KEY
 
 jobs:
-  test:
+  all:
     docker:
       - image: circleci/node:10-stretch
     steps:
       - checkout
       - run:
           name: Install dependencies
-          command: npm i
+          command: |
+            npm i
+            npm run bootstrap:ci
       - run:
           name: Lint
           command: npm run lint
+      - run:
+          name: Build
+          command: npm run build
       - run:
           name: Test
           command: npm run test
+      - run:
+          name: Configure deployment
+          command: |
+            static_s3_base="s3://cf-apps-static/apps-test-$CIRCLE_SHA1"
```

Comment:
Let's please not use the prd bucket for test deployments - we should get a second one for that.

Reply:
Makes sense.